plugins/modules/configuration_item_info.py

```diff
@@ -256,12 +273,27 @@ def run(module, table_client, attachment_client):
             module.params, "sys_id", "name", "sysparm_query", "sysparm_display_value"
         )
 
+    # default yes, only disabled if not selected in return_fields
+    query_attachments = True
```

Comment:
This is a bit confusing. I would expect here to be `False` and set to `True` is selected in the `return_fields`.

Reply:
Default behavior is to return `attachments` as return fields, we need to disable the behavior only when the `return_fields` does not select `attachments`.

Reply:
IMO, a `default behavior` is when the user does not do anything. 

In this case, the default behavior is the opposite (i.e. not getting the `attachments`)
If the user _wants_ the `attachments`, he needs to add `attachments` in the `return fields`.

Reply:
for me it's ok actually. I prefer this behavior